### PR TITLE
(1)fixed test (points changed in osm (2) updated osmnx package

### PR DESCRIPTION
**Pull request by tzufgoogle**
(1)fixed test (points changed in osm (2) updated osmnx package.
update the package to the latest version and update the environment yml file.

### CODE REVIEW

File: environment.yml [hunk 1]
```diff
@@ -414,67 +414,15 @@ dependencies:
...
-    - torchtext==0.7.0
-    - transformers==3.3.1
+    - tokenizers==0.9.4
+    - torch==1.7.1
+    - torchtext==0.8.1
+    - transformers==4.1.1
     - webcolors==1.11.1
-    - yarl==1.5.1
-prefix: /opt/conda/envs/cabby
+prefix: /home/nlp/tzufar/anaconda3/envs/cabby
```
Comment (jasonbaldridge) [inline environment.yml:427]: Can you change this prefix back?

**Review by jasonbaldridge — APPROVED** [review]
Thanks, just a minor thing to fix. Giving LGTM so you can submit.

File: environment.yml (hunk 1 above)
Comment (tzufgoogle) [reply environment.yml]: Thanks, didn't notice that. Changed it back to the previous prefix.

**Comment by tzufgoogle** [comment]
Minor conda-environment change

**Pull request merged**